guidelines/index.html

```diff
@@ -2439,9 +2447,16 @@ <h4 id="x4-1-1-parsing"><span class="secno">達成基準 4.1.1 </span>構文解
    <p>マークアップ言語を用いて実装されているコンテンツにおいては、要素には完全な開始タグ及び終了タグがあり、要素は仕様に準じて入れ子になっていて、要素には重複した属性がなく、どの ID も一意的である。ただし、仕様で認められているものを除く。
    </p>
    
-   <div class="note" id="issue-container-generatedID-26"><div role="heading" class="note-title marker" id="h-note-26" aria-level="5"><span>注記</span></div><p class="">開始及び終了タグに重要な記号が欠けている場合、例えば、タグを閉じる山括弧がない、又は属性値の引用符が一致していない場合は、完全とはいえない。
-   </p></div>
+   <div class="note" role="note" id="issue-container-generatedID-26"><div role="heading" class="note-title marker" id="h-note-26" aria-level="5"><span>注記</span></div><p class="">この達成基準は、HTML 又は XML を使用するすべてのコンテンツについて常に満たされているとみなされるべきである。</p></div>
    
+  <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記</span></div><div class="">
+      <p>この基準が作成されて以来、HTML Living Standard は、ユーザーエージェントが不完全なタグ、不正な要素のネスト、重複した属性、および一意でない ID をどのように処理しなければならないかを規定する明確な要件を採用した。 [<cite><a class="bibref" data-link-type="biblio" href="#bib-html" title="HTML Standard">HTML</a></cite>]</p>
+      
+      <p>HTML Standard はまた、これらのケースの一部を著者に対して不適合として扱うが、仕様ではユーザエージェントがこれらのケースの一貫した処理をサポートすることを要求しているため、この達成基準の目的では「これらの機能を許可する」と見なされる。実際には、この基準自体は、もはや障害のある人々に何の利益ももたらさない。</p>
```

Comment:
気になった点として、

- 冒頭のAlthoughは「……が、」だけで表現できているので、「また」はなくて良いのではないかと思います。
- authorには「コンテンツ制作者」という訳語を当てていたはず。
- 「見なす」はすぐ上では「みなす」だったので、こちらの「見なされる」もひらがなに
- 引用符付きの "allow these features" は達成基準の説明文の末尾から引用しているものなので、そこの訳は揃える必要がありそう。現状 "except where the specifications allow these features" を「ただし、仕様で認められているものを除く」としているので、「認められているもの」か。「仕様で認められているもの」まで持ってきてしまってもいいかも。

というところで、調整しつつ以下のようにしてみてはどうでしょうか。

```suggestion
      <p>HTML Standard では、これらのケースのいくつかをコンテンツ制作者に対する不適合として扱うが、仕様ではユーザエージェントがこれらのケースの一貫した処理をサポートすることを要求しているため、この達成基準の説明にある「仕様で認められているもの」とみなせる。実際には、この基準自体は、もはや障害のある人々に何の利益ももたらさない。</p>
```

Reply:
ありがとうございます。suggestを取り込みました。